components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/oauth/dao/OAuthConsumerDAOTest.java

```diff
@@ -170,4 +190,111 @@ public void testGetRequestToken() throws Exception {
+        ResultSet resultSet2 = null;
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection3 = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection3);
+
+            OAuthConsumerDAO consumerDAO = new OAuthConsumerDAO();
+            consumerDAO.createOAuthRequestToken(clientId, acc_Token, secret, callback, scope);
+
+            try {
+                statement1 = connection3.prepareStatement(GET_ID);
+                statement1.setString(1, clientId);
+                resultSet1 = statement1.executeQuery();
+                if(resultSet1.next()) {
+                     ID = resultSet1.getInt(1);
+                }
+
+                statement2 = connection3.prepareStatement(REQ_ID);
+                statement2.setInt(1, ID);
+                resultSet2 = statement2.executeQuery();
+                if(resultSet2.next()){
+                    REQ_TOKEN = resultSet2.getString(1);
+                }
+            } finally {
+                IdentityDatabaseUtil.closeAllConnections(connection3, resultSet1, statement1);
+            }
+
+            assertEquals(REQ_TOKEN, req_Token, "Checking whether the passed req_Token is set to the " +
+                    "REQ_TOKEN.");
+        }
+    }
+
+    @Test
+    public void testCreateOAuthRequestToken_02() throws Exception {
+
+        String callback_URL = null;
+        String c_Id = "fakeClientId";
+        String ac_Token = "fakeAccToken";
+        String secretFake = "fakeSecret";
+        String fakeScope = "fakeScope";
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
+        try(Connection connection3 = getConnection(DB_Name)) {
+            mockStatic(IdentityDatabaseUtil.class);
+            when(IdentityDatabaseUtil.getDBConnection()).thenReturn(connection3);
+
+            OAuthConsumerDAO consumerDAO = new OAuthConsumerDAO();
+            consumerDAO.createOAuthRequestToken(c_Id, ac_Token, secretFake, callback_URL, fakeScope);
+        }
+    }
+
+    @Test
+    public void testAuthorizeOAuthToken() throws Exception {
+
+        mockStatic(OAuthServerConfiguration.class);
+        when(OAuthServerConfiguration.getInstance()).thenReturn(mockedServerConfig);
+        PlainTextPersistenceProcessor processor = new PlainTextPersistenceProcessor();
+        when(mockedServerConfig.getPersistenceProcessor()).thenReturn(processor);
+
```

Comment:
unnecessary new line.
Instead, add a comment to indicate the separation. It will make the code easily understandable.